docs/pt/docs/tutorial/body-multiple-params.md

```diff
@@ -0,0 +1,170 @@
+
+Assim como existem uma `Query` e uma `Path` para definir dados adicionais para parâmetros de consulta e de rota, **FastAPI** provê o equivalente para `Body`.
+
+Por exemplo, extendendo o modelo anterior, você poder decidir por ter uma outra chave `importance` no mesmo corpo, além de `item` e `user`.
+
+Se você declará-lo como é, porque é um valor singular, **FastAPI** vai assumir que se trata de um parâmetro de consulta.
+
+Mas você pode instruir **FastAPI** para tratá-lo como outra chave do corpo usando `Body`:
+
+
+```Python hl_lines="23"
+{!../../../docs_src/body_multiple_params/tutorial003.py!}
+```
+
+Neste caso, **FastAPI** vai esperar um corpo como:
+
+
+```JSON
+{
+    "item": {
+        "name": "Foo",
+        "description": "The pretender",
+        "price": 42.0,
+        "tax": 3.2
+    },
+    "user": {
+        "username": "dave",
+        "full_name": "Dave Grohl"
+    },
+    "importance": 5
+}
+```
+
+Mais uma vez, ele vai converter os tipos de dados, validar, documentar, etc.
+
+## Múltiplos parâmetros de corpo e consulta
+
+Obviamente, você pode também declarar parâmetros de consulta assim que você precisar, de modo adicional a quaisquer parâmetros de corpo.
+
+Dado que, por padrão, valores singulares são interpretados como parâmetros de consulta, você não precisa explicitamente adicionar uma `Query`, você pode somente:
+
+```Python
+q: Optional[str] = None
+```
+
+como em:
+
+```Python hl_lines="28"
+{!../../../docs_src/body_multiple_params/tutorial004.py!}
+```
+
+!!! info
+    `Body` também possui todas as validações adicionais e metadados de parâmetros como em `Query`,`Path` e outras que você verá depois.
+
+
+## Requisite a indicação da chave para um único parâmetro de corpo
+
+Suponha que você tem um único `item` parâmetro de corpo a partir de um modelo Pydantic `Item`.
+
+Por padrão, **FastAPI** vai então esperar que seu conteúdo venha no corpo diretamente.
+
+Mas se você quiser que ele requisite por um JSON com uma chave `item` e dentro dele os conteúdos do modelo, como ocorre ao declarar vários parâmetros de corpo, você pode usar o parâmetro especial de `Body` chamado `embed`:
+
+```Python
+item: Item = Body(..., embed=True)
+```
+
+como em:
+
+```Python hl_lines="17"
+{!../../../docs_src/body_multiple_params/tutorial005.py!}
+```
+
+Neste caso **FastAPI** vai esperar um corpo como:
+
+```JSON hl_lines="2"
+{
+    "item": {
+        "name": "Foo",
+        "description": "The pretender",
+        "price": 42.0,
+        "tax": 3.2
+    }
+}
+```
+
+ao invés de:
+
+```JSON
+{
+    "name": "Foo",
+    "description": "The pretender",
+    "price": 42.0,
+    "tax": 3.2
+}
+```
+
+## Recapitulando
+
+Você pode adicionar múltiplos parâmetros de corpo para sua *função de operação de rota*, mesmo que a requisição possa ter somente um único corpo.
+
+E o **FastAPI** vai manipulá-los, mandar para você os dados corretos na sua função, e validar e documentar o schema correto na *operação de rota*.
```

Comment:
```suggestion
E o **FastAPI** vai manipulá-los, mandar para você os dados corretos na sua função, validar e documentar o schema correto na *operação de rota*.
```

Reply:
I think in this case "validar e documentar o schema..." is a single unity in the enumeration. If it was a list:
E o FastAPI vai:
 - manipulá-los;
 - mandar para você os dados corretos na sua função, e;
 - validar e documentar o schema correto na *operação de rota*

Reply:
Hmmm, okay